src/content/docs/analyzer/import-sorting.mdx

```diff
@@ -113,6 +113,40 @@ import "../polyfills/array/flatMap";
 import { functionThatUsesFlatMap } from "./utils.js";
 ```
 
+## Side effect imports
+
+A [side effect import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#forms_of_import_declarations) are import statements that usually don't import any name:
+
+```js
+import "./global.js"
+```
+
+Since it is hard to determine which side effects a module triggers,
+the import sorting creates a group for every side effect imports.
```

Comment:
```suggestion
Since it is hard to determine which side effects a module triggers, the import sorting creates a group for every side effect imports.
```

Sorry to be pedantic, truly, but going to a new line after a comma really hurts my eyes. It isn't about the rendered code, but the source code.